wls-vorfaelleundvorkommnisse-service/src/main/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/domain/ereignis/EreignisartEntity.java

```diff
@@ -0,0 +1,22 @@
+package de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.domain.ereignis;
+
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.rest.ereignis.EreignisartDTO;
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.service.EreignisartModel;
+
+public enum EreignisartEntity {
+    VORFALL, VORKOMMNIS;
+
+    public static EreignisartEntity fromModel(EreignisartModel model) {
+        return switch (model) {
+        case VORFALL -> VORFALL;
+        case VORKOMMNIS -> VORKOMMNIS;
+        };
+    }
+
+    public static EreignisartEntity fromDto(EreignisartDTO entity) {
+        return switch (entity) {
+        case VORFALL -> VORFALL;
+        case VORKOMMNIS -> VORKOMMNIS;
+        };
+    }
```

Comment:
das gehört in den jeweiligen Mapper oder Testdatenfactory.

Reply:
siehe commit [b1a086e](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/b1a086e3245ecd720cc3083f9e237c025716cf4d)